packages/idyll-compiler/test/test.js

```diff
@@ -1,4 +1,4 @@
-
+const inspector = require('inspector');
```

Comment:
Is this being used? Please remove if not -- `inspector` isn't in the dependencies so it is causing the tests to fail.